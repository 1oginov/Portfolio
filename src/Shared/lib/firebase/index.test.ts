import { getAuthProvider } from './index';

describe('getAuthProvider', () => {
  it('returns Google auth provider if "google" passed', () => {
    expect(getAuthProvider('google')).toHaveProperty('providerId', 'google.com');
  });

  it('throws error matching snapshot if unknown provider passed', () => {
    expect(() => getAuthProvider('unknown')).toThrowErrorMatchingSnapshot();
  });
});
