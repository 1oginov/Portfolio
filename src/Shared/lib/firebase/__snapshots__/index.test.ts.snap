// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`getAuthProvider throws error matching snapshot if unknown provider passed 1`] = `"Unknown auth provider"`;
